app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,52 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
+body: |
+  This component requires at least a non-falsey `number` attribute value passed to it, otherwise it will not render.
+accessibility_criteria: |
+  This component must:
+  - uses clear language and is positioned in a logical context on the page it's being used on
+  - communicates number value and label (if present) in a single dictation when reading with screen reader software
+shared_accessibility_criteria:
+- link
+examples:
+  default:
+    data:
+      number: 119
+  with_labels:
+    description: |
+      Labels for numbers are given inline styling but stacked using pseudo elements with display block. This is to bypass an issue with NVDA where block level elements are dictated separately.
```

Comment:
Nitpick: this description (and a few below) don't need to be formatted using `|` as it's a single line. You could just write `description: Labels for numbers are given...`